src/components/FileTreeNode/index.tsx

```diff
@@ -101,13 +105,22 @@ class FileTreeNodeBase<TreeNodeType> extends React.Component<PublicProps> {
     let linterType = null;
     if (linterMessages && hasLinterMessages) {
       linterType = findMostSevereTypeForPath(linterMessages, node.id);
+      let title = getTitleForType(linterType, isFolder);
+
+      if (
+        linterType === 'notice' &&
```

Comment:
I'm not convinced that known library messages will always have the type `notice` (I can't be sure from looking at addons-linter). I'd rather not see our code make that assumption. Can you find another way to allow more severe messages take precedence? To be clear, I do agree that a known library message should override any other message of `type=notice`.

Reply:
@kumar303 heh. the docs says it is a `notice`: https://github.com/mozilla/addons-linter/blob/5df2b4ecce7d9c9ad6abc0e6c7d1283a5b0e249c/docs/rules.md#javascript